details/operator-sdk/config/manager/manager.yaml

```diff
@@ -1,10 +1,3 @@
-apiVersion: v1
-kind: Namespace
-metadata:
-  labels:
-    control-plane: controller-manager
-  name: system
----
```

Comment:
Forgot to mention why this is being removed; it's clashing with the namespace resource in `neptune.yaml`, and probably shouldn't be in here anyhow.